pocs/utils/logger.py

```diff
@@ -45,25 +46,34 @@ def get_root_logger(profile='panoptes', log_config=None):
     # Get log info from config
     log_config = log_config if log_config else load_config('log').get('logger', {})
 
+    invoked_script = os.path.basename(sys.argv[0])
     log_dir = '{}/logs'.format(os.getenv('PANDIR', '/var/panoptes/'))
-    log_fname = 'panoptes-{}-{}.log'.format(os.getpid(), datetime.datetime.utcnow().strftime('%Y%m%dT%H%M%SZ'))
-    log_fname_generic = 'panoptes.log'
+    log_fname = '{}-{}-{}'.format(invoked_script, os.getpid(),
+                                  datetime.datetime.utcnow().strftime('%Y%m%dT%H%M%SZ'))
+    log_fname_generic = '{}'.format(invoked_script)
```

Comment:
Why is this not just log_fname_generic = invoked_script

Reply:
Ha, good point. This is why code reviews are good and speed-coding is bad. :)